.travis.yml

```diff
@@ -1,20 +1,26 @@
 ---
 sudo: false
 
-cache:
-  - pip
-
 language: python
+
 python:
   - "2.7"
   - "3.5"
 
+cache:
+  - pip
+
+before_cache:
+  - rm ~/.cache/pip/log/debug.log
+
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=utils
+
 install:
-  - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
-  # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
-  - ansible-playbook --syntax-check playbooks/byo/config.yml
-  - tox
-  - cd utils && tox
+  - if [[ "${TEST_DIR}" == "root" ]]; then tox; fi
+  - if [[ "${TEST_DIR}" == "utils" ]]; then cd utils && tox; fi
```

Comment:
I'm not able to open the Travis build logs now (infra problem), but I was wondering how this will appear.
There are 2 commands to run, and one command will always abort and the other will run tox (because we have TEST_DIR either == root or utils).

Thinking aloud... though I don't like much have a long line with an inline shell script, perhaps the two commands could be factored into one:

```yaml
script:
  - if [[ "${TEST_DIR}" == "root" ]]; then tox; else cd utils && tox; fi
```

Or even:

```yaml
env:
  - TEST_DIR=. TEST_TYPE=lint
  - TEST_DIR=. TEST_TYPE=unit
  - TEST_DIR=utils

script:
  - cd "${TEST_DIR}" && tox
```